docs/docs/spark-procedures.md

```diff
@@ -20,7 +20,7 @@ title: "Procedures"
 
 # Spark Procedures
 
-To use Iceberg in Spark, first configure [Spark catalogs](spark-configuration.md). Stored procedures are only available when using [Iceberg SQL extensions](spark-configuration.md#sql-extensions) in Spark 3.
+To use Iceberg in Spark, first configure [Spark catalogs](spark-configuration.md). Stored procedures are only available when using [Iceberg SQL extensions](spark-configuration.md#sql-extensions) in Spark.
```

Comment:
configuring the extension is not necessary for Spark 4.

Reply:
For SPARK-53523, we may ignore it until Spark 4.1.0 is released and supported in Iceberg.